### PR TITLE
CNDIT-1039 : Update service to be dockerized and deployable.

### PR DESCRIPTION
**Pull request by alog-enquizit**
- Convert the application.properties to yaml.
- Added Liquibase support for database schema management
- Updating the main service Dockerfile to fix containerization issues.
- Added Dockerfile for restoring the NBS databases
- Added the docker-compose.yml to build - restore nbs database, build zookeper, build kafka broker and the etl service
- Added a build script to invoke the service build sequence

### CODE REVIEW

File: etl-data-pipeline/src/main/resources/application.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+spring:
+  kafka:
+    producer:
+      value-serializer: org.springframework.kafka.support.serializer.JsonSerializer
+      key-serializer: org.springframework.kafka.support.serializer.StringOrBytesSerializer
+    stream:
+      input:
```
Comment (upasanapattnaik-eq) [inline etl-data-pipeline/src/main/resources/application.yaml:7]: If we want to add new topics, does this get updated while the application is in service or it needs to be defined before every docker build?

File: etl-data-pipeline/src/main/resources/db/sp-organization-001.sql [hunk 2]
```diff
@@ -1,4 +1,4 @@
-CREATE PROCEDURE [dbo].[sp_Organization]
+CREATE or Alter PROCEDURE [dbo].[sp_Organization]
```
Comment (upasanapattnaik-eq) [inline etl-data-pipeline/src/main/resources/db/sp-organization-001.sql:1]: This will be maintained by liquibase, correct?

**Review by upasanapattnaik-eq — APPROVED** [review]
LGTM. Thank you for setting this up!

File: etl-data-pipeline/src/main/resources/application.yaml (hunk 1 above)
Comment (alog-enquizit) [reply etl-data-pipeline/src/main/resources/application.yaml:7]: If you are going to use the consolidated docker images i.e., zookeeper, kafka and the etl service built by the docker compose, then yeah we have to rebuild everytime when we have add a new topic or update the code.

File: etl-data-pipeline/src/main/resources/db/sp-organization-001.sql (hunk 2 above)
Comment (alog-enquizit) [reply etl-data-pipeline/src/main/resources/db/sp-organization-001.sql:1]: Yeah if you have to make any changes going forward, you will have to create a new revision file such as:
sp-organization-002.sql

**Comment by alog-enquizit** [comment]
> LGTM. Thank you for setting this up!

Thanks @upasanapattnaik-eq !

**Pull request merged**